docs/install/install_options/install_options.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Options
+---
+
+This page focuses on the configuration options available when setting up RKE2:
+
+- [Configuring the installation script](#configuring-the-installation-script)
+- [Configuring RKE2 server nodes](#configuring-rke2-server-nodes)
+- [Configuring RKE2 agent nodes](#configuring-rke2-agent-nodes)
+- [Using the configuration file](#configuration-file)
+- [Configuring when running the binary directly](#configuring-when-running-the-binary-directly)
+
+The primary way to configure RKE2 is through its [config file](#configuration-file). Command line arguments and environment variables are also available, but RKE2 is installed as a systemd service and thus these are not as easy to leverage.
+
+### Configuring the Installation Script
+
+As mentioned in the [Quick-Start Guide](../../install/quickstart.md), you can use the installation script available at https://get.rke2.io to install RKE2 as a service.
+
+The simplest form of this command is as follows:
+```sh
+curl -sfL https://get.rke2.io | sh -
+```
+
+When using this method to install RKE2, the following environment variables can be used to configure the installation:
+
+| Environment Variable | Description |
+|-----------------------------|---------------------------------------------|
+| <span style="white-space: nowrap">`INSTALL_RKE2_VERSION`</span> | Version of RKE2 to download from GitHub. Will attempt to download from the latest channel if not specified. |
```

Comment:
You'll need to remove the `<span>` tags throughout install_options.md as I didn't get that in before you ran with what I had.